force-app/main/default/lwc/newProgramEngagement/newProgramEngagement.js

```diff
@@ -31,8 +32,14 @@ import newContact from "@salesforce/label/c.New_Contact";
 import cancelAndBack from "@salesforce/label/c.Cancel_and_Back";
 import cantFindContact from "@salesforce/label/c.Cant_Find_Contact";
 
-const CREATE_PROGRAM_ENGAGEMENT_FIELD_SET = "CreateProgramEngagement";
-const CREATE_CONTACT_FIELD_SET = "CreateContact";
+const ACTIVE = "Active";
+const ENROLLED = "Enrolled";
+const ALLOWED_STAGES = [ACTIVE, ENROLLED];
```

Comment:
Could this be set up as a enum-like structure? 

```
const ACTIVE_ENGAGEMENT_STAGES_ENUM = Object.freeze({
    ACTIVE:  "Active",
    ENROLLED: "Enrolled"
});
```

Reply:
yea but thinking it's not really necessary here since this is temporary?

Reply:
That makes perfect sense, good perspective!